src/main/java/duke/commands/AddCommand.java

```diff
@@ -0,0 +1,88 @@
+package duke.commands;
+
+import duke.exceptions.CommandParamException;
+import duke.exceptions.DukeFileException;
+import duke.exceptions.EmptyDescriptionException;
+import duke.exceptions.UnknownCommandException;
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.ToDo;
+import duke.TaskList;
+import duke.Storage;
```

Comment:
Is the lexicographical order wrong here? Perhaps it should be something like this : 
```
//other imports
duke.execeptions.UnknownCommandException;
duke.Storage;
duke.task.task.ToDo;
```